lib/core/options.js

```diff
@@ -96,16 +96,10 @@ function resolveTracingOptions (tracing, config) {
   /** @type {NonNullable<import('../../types/sentry').TracingConfiguration>} */
   const tracingOptions = merge(
     {
-      tracesSampleRate: 1.0,
-      vueOptions: {
-        tracing: true,
-        tracingOptions: {
-          hooks: ['mount', 'update'],
-          timeout: 2000,
-          trackComponents: true,
-        },
-      },
-      browserOptions: {},
+      hooks: ['mount', 'update'],
+      timeout: 2000,
+      trackComponents: true,
+      tracePropagationTargets: ['localhost', /^\//],
```

Comment:
This seems to be the default value. Is it necessary to define it?

And related to that, why are we only passing `tracePropagationTargets` to `new BrowserTracing`? Shouldn't we allow the user to override all the options that it supports?

Reply:
You're right. It's now possible by passing the options into `browserTracing` key in the tracing options.